tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "types": ["jest"],
     "typeRoots": ["./types", "./node_modules/@types"],
     "noErrorTruncation": true,
+    "skipLibCheck": true,
```

Comment:
What's this for?

According to [here](https://www.typescriptlang.org/tsconfig#skipLibCheck): "This can save time during compilation at the expense of type-system accuracy. For example, two libraries could define two copies of the same type in an inconsistent way."

Reply:
I had an issue with TSServer giving error about something on `node_modules`. This is the default on `@tsconfig/node18/tsconfig.json` that we use throughout, just not on the root.

Also, the argument about the type is not necessarily valid, since that would be perfectly fine, each type on its own "namespace"/lib.